spec/controllers/destinations_controller_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe DestinationsController, type: :controller do
+
+    let(:valid_destination_attributes) {
+        { :city => 'San Francisco', :country => 'USA', :currency => 'USD' }
+    }
+
+    let(:invalid_destination_attributes) {
+        { :city => 'San Francisco', :country => '', :currency => 'USDT' }
+    }
+
+    let!(:destination_data) { create(:destination) }
+
+    describe 'Destinations /index' do
+        before(:each) do
+            get :index
+        end
+
+        it 'renders the index template' do
+            expect(response).to render_template('index')
```

Comment:
render_template we don't care about rendering files.
Here we should strictly test the response after a get, post method.
You should have access on response.body after you make a get method in the it block.